airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/manifest.yaml

```diff
@@ -0,0 +1,465 @@
+version: 0.61.2
+type: DeclarativeSource
+check:
```

Comment:
Do we need this as the method `check_connection` is overridden?

Reply:
No, because we are adding additional behavior after the default low-code check connection logic.